Mathlib/RingTheory/Localization/AtPrime/Basic.lean

```diff
@@ -181,22 +180,18 @@ variable (I : Ideal R) [hI : I.IsPrime]
 variable {I}
 
 /-- The unique maximal ideal of the localization at `I.primeCompl` lies over the ideal `I`. -/
-theorem AtPrime.comap_maximalIdeal :
+nonrec theorem AtPrime.comap_maximalIdeal :
```

Comment:
I think giving the full name is clearer. It just doesn't need a comment. (Personally I would avoid `nonrec` as much as possible for readability.)